site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2463,6 +2465,52 @@ public function getGradedComponentsCountByGradingSections($g_id, $sections, $sec
         return $return;
     }
 
+
+    /**
+     * Returns an array of Verified components for each Section
+     *
+     * @param  array<int>  $sections
+     * @return array<int|string,int>
+     */
+    public function getVerifiedComponentsCountByGradingSections(string $g_id, array $sections, string $section_key, bool $is_team): array {
+        $unit = "users";
+        $id = "user_id";
+        if ($is_team) {
+            $unit = "gradeable_teams";
+            $id = "team_id";
+        }
+
+        if (! in_array($section_key, ["registration_section", "rotating_section"], true)) {
+            return [];
+        }
+
+        $this->course_db->query(
+            "
+            SELECT 
+                $unit.$section_key, COUNT($unit.$id) as verified_components_count
+            FROM 
+                gradeable_data as gd
+                JOIN gradeable_component_data as gcd ON (gd.gd_id = gcd.gd_id)
+                JOIN $unit ON (gd.gd_$id = $unit.$id)
+            WHERE
+                gd.g_id = ?
+                AND CAST ($unit.$section_key AS TEXT) = ANY (ARRAY[" . implode(',', array_fill(0, count($sections), '?')) . "])
```

Comment:
A few comments about this line:
1. The `implode(...)` can be replaced with our `$this->createParameterList()` helper.
2. Is the a reason why you can't use `IN` instead of `= ANY`?

Reply:
90927284a67635ab386581c2dbbc09175bc4ebb7